README.md

```diff
@@ -82,7 +82,8 @@
 <!-- GETTING STARTED -->
 ## 💻 Getting Started <a name="getting-started"></a>
 
-> Open the index.html file to view the contents 
+> Click <a href="https://yosefgeda.github.io/My-Portfolio/">HERE</a> to view the page.
+ 
```

Comment:
- [ ] [OPTIONAL] I think it would be great if you put the link under the section `Live Demo` to make it easier for visitors to notice it quickly. 😎️

![Screenshot from 2023-01-24 10-50-43](https://user-images.githubusercontent.com/17243325/214239178-35fff637-b142-4cab-abf1-3f2f0845b983.png)